data/themes/default/style.css

```diff
@@ -532,6 +532,22 @@ TimeLine {
 	font-size: 8px;
 }
 
+TimeLineWidget {
+	/* font-size only supports px and pt. */
+	font-size: 7pt;
+	/* lengths also support em. This will make sure that the height
+	   will always change in the same proportion as the font size
+	   defined above.
+	   If you want a fixed size set min and max to the same value. */
+	min-height: 1.5em;
+	max-height: 1.5em;
+
```

Comment:
I have reinstated the old look. I have to admit that I like the flat look better but that's something for theme designers I guess. :)

I have also removed the `TimeLine` entry from the `style.css`.